Symfony/CS/Fixer/PSR2/SwitchCaseSpaceFixer.php

```diff
@@ -38,7 +38,7 @@ public function fix(\SplFileInfo $file, $content)
                     ++$ternariesCount;
                 }
 
-                if ($tokens[$colonIndex]->equals(':')) {
+                if ($tokens[$colonIndex]->equalsAny(array(':', ';'), false)) {
```

Comment:
No, a case-sensitive (`true`) check is faster than a case-insensitive (`false`).
But in this case we only have non-array tokens, so the parameter is not used:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.12/Symfony/CS/Tokenizer/Token.php#L117